edxval/tests/test_api.py

```diff
@@ -1099,3 +1100,22 @@ def test_remove_video_for_course(self):
         # verify that video for other course has the correct info
         video_info = {key: videos[0][key] for key in constants.VIDEO_DICT_FISH}
         self.assertEqual(video_info, constants.VIDEO_DICT_FISH)
+
+
+class GetVideoStatusUpdateTest(TestCase):
+    """
+    Tests to check `update_video_status` function works correctly
+    """
+    @patch('edxval.models.logger')
+    def test_update_video_status_logging(self, mock_logger):
+        """
+        Tests video status update logging
+        """
+        video = Video.objects.create(**constants.VIDEO_DICT_FISH)
+        mock_logger.info.assert_called_with(
+            'Video created with id [%s] and status [%s]', video.edx_video_id, constants.VIDEO_DICT_FISH.get('status')
+        )
```

Comment:
You may want to also have a negative test here - to make sure the "Status changed to" logging does *not* happen at the time the `Video` object is created.